tests/evaluate/test_evaluation.py

```diff
@@ -568,6 +568,26 @@ def test_pandas_df_regressor_evaluation(linear_regressor_model_uri):
         assert v == saved_metrics[k]
 
 
+def test_spark_df_regressor_evaluation(linear_regressor_model_uri):
+    data = sklearn.datasets.load_diabetes()
+    spark = SparkSession.builder.master("local[*]").getOrCreate()
+    panda_df = pd.DataFrame(data.data, columns=data.feature_names)
+    panda_df["label"] = data.target
+    spark_df = spark.createDataFrame(panda_df)
+    with mlflow.start_run() as run:
+        eval_result = evaluate(
+            linear_regressor_model_uri,
+            data=spark_df,
+            targets="label",
+            model_type="regressor",
+            evaluators=["default"],
+        )
+    _, saved_metrics, _, _ = get_run_data(run.info.run_id)
+
+    for k, v in eval_result.metrics.items():
+        assert v == saved_metrics[k]
+
```

Comment:
can we check that the dataset is logged to this run? should be available from `run.inputs.dataset_inputs`. and can we also check information about that dataset such as the source?

Reply:
done!